### ISSUE TITLE
Aspire should get removed from the VMR

### ISSUE THREAD

**Issue by ViktorHofer** [dotnet/source-build#4735]
With https://github.com/dotnet/aspire/commit/9f212d861ab8dca5f62ca908120e7040df0c2be3#diff-366b25472963c6b33188d84139f6c8997298c5adc857a2624ab0d26a2d87baf0L10, aspire doesn't produce the workload manifest or any other artifact anymore inside the VMR.

We should be able to remove it from the VMR.

cc @joperezr

**Comment by MichaelSimons** [comment]
The aspire workload still getting bundled by the SDK.  Note the reference is an out of support 8.n version rather than 9.0.  There was a thread with the aspire folks on this.  IIRC there wasn't a decision on when they would remove this.  @joperezr has there been any decision on this?
 
SDK workload reference: https://github.com/dotnet/sdk/blob/main/src/Installer/redist-installer/targets/BundledManifests.targets#L19

**Comment by joperezr** [comment]
Not yet, but if I had to guess, I'd assume this would be done for .net 10.

**Comment by ViktorHofer** [comment]
@joperezr @danmoseley do you have an update on when the Aspire workload will get removed from the SDK? Asking as this is blocking https://github.com/dotnet/source-build/issues/4359 (see https://github.com/dotnet/sdk/pull/47254 for more details).

**Comment by joperezr** [comment]
Same answer as Nov, my guess would be .NET 10. At some point later this year Aspire is planning to ship Aspire 10.0 previews, and I think that this will be the time to take a look at this.

**Comment by dviererbe** [comment]
Hey o/ I am looking to resolve #5049, but Michal mentioned this issue. Resolving this issue would eventually also resolve #5049.

> do you have an update on when the Aspire workload will get removed from the SDK? 
> > Same answer as Nov, my guess would be .NET 10.

**Question:** Could we get a commitment that Aspire will be removed until the .NET 10 GA? / Who do we need to talk to to get a definitive answer?

**Question:** Will the Aspire workload also be removed from the VMR for .NET 9? My assumption would be no.

**Comment by joperezr** [comment]
Yes, we will delete it by .NET 10. I think we'd still need to have a workload breadcrum (like the targets and manifst files) but they will just be empty.
